tests/openedx_learning/core/components/test_api.py

```diff
@@ -0,0 +1,439 @@
+"""
+Basic tests of the Components API.
+"""
+from datetime import datetime, timezone
+
+from django.core.exceptions import ObjectDoesNotExist
+
+from openedx_learning.core.components import api as components_api
+from openedx_learning.core.contents import api as contents_api
+from openedx_learning.core.publishing import api as publishing_api
+from openedx_learning.lib.test_utils import TestCase
+
+
+class TestPerformance(TestCase):
+    """
+    Performance related tests to make sure we don't get n + 1 queries.
+    """
+    @classmethod
+    def setUpTestData(cls) -> None:
+        """
+        Initialize our content data (all our tests are read only).
```

Comment:
Nit: I don't think the tests are actually read only? They're doing stuff like `create_component_and_version`. Maybe you mean "None of the tests modify the Learning Package itself" ?

Reply:
Oops. Yeah, sloppy wording on my part, will fix, ty.